theme/json/posts.json

```diff
@@ -1,29 +1,38 @@
 [
 	{
+		"id": "101",
```

Comment:
id's should start at 0, for future feature to work.

Reply:
done